dm/worker/meta_test.go

```diff
@@ -69,6 +76,16 @@ func testSetUpDB(c *C) (*leveldb.DB, string) {
 		Task:  testTask2Bytes,
 	}
 
+	testTask3Str, err := testTask3.Toml()
```

Comment:
why add it?

Reply:
add one more task meta to test _delete by batch_ when clearing task meta.